.github/workflows/ci.yml

```diff
@@ -64,14 +65,15 @@ jobs:
 
   test:
     name: test
-    runs-on: ubuntu-latest
 
 # begin macro
 
+    runs-on: ${{ matrix.platform }}
     strategy:
       fail-fast: false
       matrix:
-        node-version: [12.17.0, 12.x, 14.x]
+        node-version: [12.x]
```

Comment:
Why are we reverting back to node 12 only?

Reply:
This is in error. Posted a revision that makes forward progress instead, by adding `16.x` and using only `16.x` in the tests that have a single element in their version matrix.